Product/Production/Services/AuditRepositoryCore/src/main/java/gov/hhs/fha/nhinc/auditrepository/nhinc/AuditRepositoryOrchImpl.java

```diff
@@ -93,34 +96,8 @@ public AuditRepositoryOrchImpl() {
     public AcknowledgementType logAudit(LogEventSecureRequestType mess, AssertionType assertion) {
 
         AcknowledgementType response = new AcknowledgementType();
-        AuditRepositoryRecord auditRec = new AuditRepositoryRecord();
-
-        EventIdentificationType eventIdentification = mess.getAuditMessage().getEventIdentification();
-
-        auditRec.setUserId(null);
-
-        String eventCommunityId = mess.getRemoteHCID();
-        LOG.info("auditSourceID : " + eventCommunityId);
-        if (NullChecker.isNotNullish(eventCommunityId)) {
-            auditRec.setRemoteHcid(eventCommunityId);
-        } else {
-            auditRec.setRemoteHcid("");
-        }
-
-        auditRec.setDirection(mess.getDirection());
-        auditRec.setMessage(getBlobFromAuditMessage(mess.getAuditMessage()));
-
-        XMLGregorianCalendar xMLCalDate = eventIdentification.getEventDateTime();
-        if (xMLCalDate != null) {
-            auditRec.setEventTimeStamp(convertXMLGregorianCalendarToDate(xMLCalDate));
-        }
-
-        auditRec.setEventType("servicetype");
-        auditRec.setOutcome(0);
-        auditRec.setEventId("EventId");
-
         List<AuditRepositoryRecord> auditRecList = new ArrayList<>();
-        auditRecList.add(auditRec);
+        auditRecList.add(createDBAuditObj(mess, assertion));
```

Comment:
Do we use this list variable to set any other value? Instead of creating this variable can we directly convert this to arraylist insertAuditRepository method?